ligo/stablecoin/operator.ligo

```diff
@@ -12,7 +12,7 @@
  * Validates the operators for the given transfer batch
  * and the operator storage.
  *)
-function is_approved_operator
+[@inline] function is_approved_operator
```

Comment:
This change seems to be misplaced (it's in a wrong commit).

Reply:
Yeah I put it there because I checked the gas costs only with the other change in this commit.

However it is likely is an improvement on its own as well, moved it to the previous commit now.